RecoEgamma/EgammaTools/python/egammaObjectModificationsInMiniAOD_cff.py

```diff
@@ -107,15 +107,37 @@ def setup_mva(val_pset,cat_pset,prod_name,mva_name):
     ecalRecHitsEE = cms.InputTag("reducedEgamma","reducedEERecHits"),
 )
 
+#############################################################
+# 8X legacy needs an extra Et scale systematic
+# due to an inflection around 45 GeV which is handled as a
+# patch on top of the standard scale and smearing systematics
+#############################################################
+from RecoEgamma.EgammaTools.calibratedEgammas_cff import ecalTrkCombinationRegression
+egamma8XLegacyEtScaleSysModifier = cms.PSet(
+    modifierName = cms.string('EGEtScaleSysModifier'),
+    epCombConfig = ecalTrkCombinationRegression,
+    uncertFunc = cms.PSet(
+        name = cms.string("UncertFuncV1"),
+        lowEt = cms.double(43.5),
+        highEt = cms.double(46.5),
+        lowEtUncert = cms.double(0.002),
+        highEtUncert = cms.double(-0.002)
+        )
+    )
+
 def appendReducedEgammaEnergyScaleAndSmearingModifier(modifiers):
     modifiers.append(reducedEgammaEnergyScaleAndSmearingModifier)
 
 def prependEgamma8XObjectUpdateModifier(modifiers):
     modifiers.insert(0,egamma8XObjectUpdateModifier)
 
+def appendReducedEgammaEnergyScaleAndSmearingModifierWith8XLegacyEtSys(modifiers):
+    modifiers.append(reducedEgammaEnergyScaleAndSmearingModifier)
```

Comment:
it seems more practical to call this method ```appendEgamma8XLegacyAppendableModifiers``` or something similarly generic.
This is cleaner than having to deal with ever-growing method name that collates what's done inside.

I can imagine that more updates can happen in the next year or so, until we can advance to a new rereco of the 2016 and abandon the 80XLegacy.

Reply:
agreed